### PR TITLE
sys-fs/zfs{,-kmod}: add 2.2.7

### PR DESCRIPTION
**Pull request by BluCobalt**
ZFS 2.2.7 for gentoo

---

Please check all the boxes that apply:

- [x] I can submit this contribution in agreement with the [Copyright Policy](https://www.gentoo.org/glep/glep-0076.html#certificate-of-origin).
- [x] I have certified the above via adding a `Signed-off-by` line to *every* commit in the pull request.
- [x] This contribution has not been created with the assistance of Natural Language Processing artificial intelligence tools, in accordance with the [AI policy](https://wiki.gentoo.org/wiki/Project:Council/AI_policy).
- [x] I have run `pkgcheck scan --commits --net` to check for issues with my commits.

Please note that all boxes must be checked for the pull request to be merged.

### CODE REVIEW

File: sys-fs/zfs-kmod/zfs-kmod-2.2.7.ebuild [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+
+EAPI=8
+
+MODULES_INITRAMFS_IUSE=+initramfs
+inherit autotools flag-o-matic linux-mod-r1 multiprocessing
+
+DESCRIPTION="Linux ZFS kernel module for sys-fs/zfs"
+HOMEPAGE="https://github.com/openzfs/zfs"
+
+MODULES_KERNEL_MAX=6.10
```
Comment (thesamesam) [inline sys-fs/zfs-kmod/zfs-kmod-2.2.7.ebuild:12]: Isn't it 6.12?

Comment (BluCobalt) [reply sys-fs/zfs-kmod/zfs-kmod-2.2.7.ebuild:13]: Yes. Let me fix that...

Comment (BluCobalt) [reply sys-fs/zfs-kmod/zfs-kmod-2.2.7.ebuild:13]: Jeez, sorry. My git skills are not very good, and my attempt to amend the commit left some weird artifact in the file.

**Comment by thesamesam** [comment]
Thanks, I'll squash it on merging.

**Comment by thesamesam** [comment]
The minimum version was also wrong.

**Pull request closed without merging**